pylabrobot/heating_shaking/hamilton.py

```diff
@@ -0,0 +1,81 @@
+from pylabrobot.heating_shaking.backend import HeaterShakerBackend
+from pylabrobot.machines.backends.usb import USBBackend
+from enum import Enum
+
+class PlateLockPosition(Enum):
+  LOCKED = 1
+  UNLOCKED = 0
+
+class Command(Enum):
+  INITIALIZE_PLATE_LOCK = "LI"
+  MOVE_PLATE_LOCK = "LP"
+  SHAKE = "SB"
+  STOP_SHAKING = "SC"
+  WAIT_FOR_STOP = "SW"
+
+class RotationDirection(Enum):
+  CLOCKWISE = 0
+  COUNTER_CLOCKWISE = 1
+
+class HamiltonHeatShaker(HeaterShakerBackend, USBBackend):
+  """
+  Backend for Hamilton Heater Shaker devices connected through
+  an HSB
+  """
+
+  def __init__(self, shaker_index:int, id_vendor:int = 2223, id_product:int = 32770) -> None:
```

Comment:
I found these numbers by trial and error, not sure if there is a better way.

Reply:
Check lsusb

Reply:
yea, I think that's what I mean by trial and error. I found the object then retrieved the values from it.

Reply:
perhaps we can keep these in hex?

it's very likely to be fixed for a given device (it's been the same for all STARs for example), so fine to hard code like this.